lib/firenest/topology/erlang.ex

```diff
@@ -115,13 +107,14 @@ defmodule Firenest.Topology.Erlang do
     end
   end
 
-  def monitor(topology, node, name) do
-    if node == Kernel.node() or node in nodes(topology) do
-      Process.monitor({name, node})
-    else
-      ref = make_ref()
-      send(self(), {:DOWN, ref, :process, {name, node}, :noconnection})
-      ref
+  def sync_named(topology, pid, timeout \\ 5000) when is_pid(pid) do
+    case Process.info(pid) do
```

Comment:
I think that should be `Process.info(pid, :registered_name)`

Reply:
Good catch.